### ISSUE TITLE
Fix Buttons for Navigating Item Menus

### ISSUE THREAD

**Issue by Ninja9000** [hackagotchi/hackagotchi#100]
The buttons that let you move through pages of items and gotchi in your inventory, next and previous (?), do not work. Make them work.

**Comment by lukec11** [comment]
@cedric-h do we know if this is still an issue past the pagination update?

**Comment by nzeng8** [comment]
I think this works for the crafting, not sure about anything else...

**Comment by lukec11** [comment]
I don't know that gotchi are paginated currently? That's why I'm not sure about this issue

**Comment by nzeng8** [comment]
Yeah, this is broken for the market, the next page button does not do anything
